vegassist.js

```diff
@@ -12,7 +12,15 @@ var DELAYTIME = 1000 * 60 * 2;
 // Load filters from all files in the filters directory
 var settingsFilteredTerms = settings.FILTERED_TERMS || [];
 var settingsFilters = settings.FILTERS || [];
-var filter = new TweetFilter('filters', settingsFilteredTerms, settingsFilters);
+
+var handle = settings.HANDLE || "";
+if (handle.length > 0){
+	if (handle.indexOf('@') != 0)
+		handle = '@' + handle;
+	util.track(handle);
```

Comment:
It will pull tweets that don't match other filters, as in test 3 above. This allows for recognition of tweets like the one noted in the issue: https://twitter.com/Floridaboy_john/status/671113287745085440